### PR TITLE
Initial implementation of Nightscout service

### PR DESCRIPTION
**Pull request by darinkrauss**
- Implement Service and ServiceUI protocols
- Implement RemoteData protocol
- Connect with Nightscout third-party service

https://tidepool.atlassian.net/browse/LOOP-22

### CODE REVIEW

**Review by nhamming — DISMISSED** [review]
LGTM

**Review by nhamming — DISMISSED** [review]
+1

**Review by nhamming — DISMISSED** [review]
+1

File: Common/LocalizedString.swift [hunk 1]
```diff
@@ -0,0 +1,23 @@
+//
+//  LocalizedString.swift
+//  NightscoutService
+//
+//  Created by Darin Krauss on 6/21/19.
+//  Copyright © 2019 LoopKit Authors. All rights reserved.
```
Comment (ps2) [inline Common/LocalizedString.swift:6]: Thanks for setting up the Copyright as LoopKit Authors for this. 👍

**Review by ps2 — APPROVED** [review]
Looks great!

File: NightscoutServiceKitUI/NightscoutServiceTableViewController.swift [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+extension AuthenticationTableViewCell: NibLoadable {}
+
+extension TextButtonTableViewCell: IdentifiableClass {}
+
+fileprivate extension UIAlertController {
+
+    convenience init(serviceDeletionHandler handler: @escaping () -> Void) {
+        self.init(
+            title: nil,
+            message: NSLocalizedString("Are you sure you want to delete this service?", comment: "Confirmation message for deleting a service"),
```
Comment (ps2) [inline NightscoutServiceKitUI/NightscoutServiceTableViewController.swift]: NSLocalizedString -> LocalizedString

**Review by ps2 — CHANGES_REQUESTED** [review]
Looks great!

**Review by ps2 — APPROVED** [review]
🎉 LGTM!

**Pull request merged**